packages/dds/tree/src/simple-tree/core/types.ts

```diff
@@ -24,9 +24,27 @@ import { isFlexTreeNode, type FlexTreeNode } from "../../feature-libraries/index
  */
 export type Unhydrated<T> = T;
 
+/**
+ * Data included for {@link TreeChangeEvents.nodeChanged}.
+ * @public
+ */
+export interface NodeChangedData {
+	/**
+	 * When the node changed is an object or Map node, this lists all the properties which changed.
+	 * @remarks
+	 * This only includes changes to the node itself (which would trigger {@link TreeChangeEvents.nodeChanged}).
+	 */
+	readonly changedProperties?: ReadonlySet<string>;
```

Comment:
Presumably `undefined` here is equivalent to an empty set? Might be worth calling that out explicitly.

Reply:
no, not equivalent. I'll update the docs.

Reply:
updated